### ISSUE TITLE
Ordering contacts in ContactSelection has no effect in FE

### ISSUE THREAD

**Issue by berkyl** [sulu/sulu#5857]
| Q | A
| --- | ---
| Sulu Version | 2.2.3
| PHP Version | 7.4.11
| DB Version | MySQL 5.7

# Actual Behavior

When you order the contacts in a ContactSelection in admin, save the changes and publish it, the order remains the same in Frontend (ordered by id)

# Expected Behavior

The list of contacts should be received in the same order as in Admin

# Steps to Reproduce

Add a contact_selection property to a block. Select a couple contacts and grab them around to order. Afterwards save and publish the changes. 
Retrieve the contacts in twig.

# Possible Solutions

The contacts are ordered by `id` in `src/Sulu/Bundle/ContactBundle/Entity/ContactRepository.php` on *line 148*.
Removing the orderBy doesn't help since the list is still ordered by occurence in the database. 
Instead add 

```
->add("orderBy", "FIELD(u.id, :ids)");
```
which requires the `FIELD` extension for mysql provided by [beberlei/DoctrineExtensions](https://github.com/beberlei/DoctrineExtensions) that needs to be registered in `doctrine.yaml`:
```
dql:
            string_functions:
                field: DoctrineExtensions\Query\Mysql\Field
```

**Comment by niklasnatter** [comment]
Thanks for reporting! I think we should sort the items that are passed to the twig template in the `ContactSelection` (and also the `AccountSelection`) like we do it in the Demo: 

https://github.com/sulu/sulu-demo/blob/0a255fba2ea9560ce87a4d622dafdf76b83fb088/src/Content/Type/AlbumSelection.php#L36-L39

**Comment by alexander-schranz** [comment]
This was fixed by #5873 in will be part of the next patch release.
